js/admission.js

```diff
@@ -0,0 +1,134 @@
+
+
+$(document).ready(function(){
+    const noResultBlock= document.querySelector(".noResultBlock");
+    loadData();
+    function loadData(){
+        courseData = getData();
+        populateUI(courseData);
+    }
+    function getData(){
+        var courses = [
+            {id:1, title: "Masters Program in Computer Science", period: "Summer 2021", level: "Master's", subjectArea: "Computer Science", paceOfStudy: "Full-time", teachingForm: "Program", applicationCode: "EUA-123"},                 
+            {id:2, title: "Masters Program in Software Engineering", period: "Summer 2021", level: "Master's", subjectArea: "Computer Science", paceOfStudy: "Full-time", teachingForm: "Program", applicationCode: "EUA-124"},
+            {id:3, title: "Masters Program in Telecommunication Systems", period: "Autumn 2021", level: "Master's", subjectArea: "Computer Science", paceOfStudy: "75%", teachingForm: "Program", applicationCode: "EUA-125"},
+            {id:4, title: "User Experience Design", period: "Summer 2021", level: "Bachelor's", subjectArea: "Computer Science", paceOfStudy: "75%", teachingForm: "Course", applicationCode: "EUA-126"},
+            {id:5, title: "Evolutionary Biology", period: "Autumn 2021", level: "Master's", subjectArea: "Biology", paceOfStudy: "Full-time", teachingForm: "Course", applicationCode: "EUA-127"},
+            {id:6, title: "Cell and Developmental Biology", period: "Autumn 2021", level: "Bachelor's", subjectArea: "Biology", paceOfStudy: "Full-time", teachingForm: "Program", applicationCode: "EUA-128"},
+            {id:7, title: "Human Physiology", period: "Summer 2021", level: "Beginner", subjectArea: "Biology", paceOfStudy: "Full-time", teachingForm: "Course", applicationCode: "EUA-129"}
+        ];
+
+        return courses;
+    }
+
+    function populateUI(data){
+        $("#search-result").empty();
+        data.forEach(course => {
+            var html = `<div class="searchresult-card">
+            <div class="resultcard-header">
```

Comment:
try to keep similar pieces of code consistent, i.e. below your html is formatted with spaces and here you have no spaces at the lines beginning. better to stick to one option